docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,265 @@
+# Using a DB with concord
+
+Sometimes, when you are developing a bot command, you may wish to save some data that will be not deleted across bot restarts. This is where a database comes in. It can save data, and, don't worry, it won't be deleted if you restart your bot.
```

Comment:
```suggestion
Sometimes, when you are developing a bot command, you may wish to save some data that will not be erased once the bot restart (aka persistent data). This is where a database comes in - it can save data, and don't worry, it won't be erased if you restart your bot.
```